### PR TITLE
moyo

### PR DESCRIPTION
**Pull request by PastushenkoVik**
- [UPDATE DEMO LINK](https://pastushenkovik.github.io/layout_moyo-header/)
- [UPDATE TEST REPORT LINK](https://pastushenkovik.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+.logo-box img {
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (irenhh) [inline src/style.css]: no need to apply align-items here

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  padding: 0;
```
Comment (irenhh) [inline src/style.css]: the same with paddings

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-of-type(1) {
+  margin-right: 0;
+}
+
+.nav-link {
+  font-family: "Roboto";
+  font-style: normal;
```
Comment (irenhh) [inline src/style.css]: why do you specify font-style here?

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  font-weight: 500;
+  font-size: 12px;
+
+  display: inline-block;
+  height: 60px;
```
Comment (irenhh) [inline src/style.css]: use either height or line-height here

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  line-height: 60px;
+  white-space: nowrap;
```
Comment (irenhh) [inline src/style.css]: you really need it here?

File: src/style.css (hunk 2 above)
Comment (PastushenkoVik) [reply src/style.css]: npm run css:reference
не проходит

File: src/style.css (hunk 5 above)
Comment (irenhh) [reply src/style.css]: that was about white-space

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+.logo-box img {
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+
```
Comment (irenhh) [inline src/style.css]: don't leave blank line in the middle of the block

File: src/style.css (hunk 2 above)
Comment (PastushenkoVik) [reply src/style.css]: We talked about default padding

File: src/index.html [hunk 7]
```diff
@@ -9,11 +9,57 @@
...
-  <title>Document</title>
+  <title>Moyo</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+  <header>
+    <a href="#" class="logo-box"><img src="./images/logo.png" alt="MOYO"></a>
```
Comment (danheim) [inline src/index.html]:
```suggestion
    <a href="#" class="logo-box">
      <img src="./images/logo.png" alt="MOYO">
    </a>
```

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+.nav-item {
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-of-type(1) {
+  margin-right: 0;
+}
+
+.nav-link {
+  font-family: "Roboto";
```
Comment (danheim) [inline src/style.css]: You can write it without brackets.
Which family will be displaying if google fonts not responding?
You should add additional families like `Arial` or `sens-serif`.

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+  color: #000;
+  text-decoration: none;
+  position: relative;
+}
+
+.nav-link.is-active {
+  color: #00acdc;
+}
+
+.nav-link.is-active:after {
```
Comment (danheim) [inline src/style.css]: `::after`

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+  left: 0;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.nav-item.is-active:hover:after {
```
Comment (danheim) [inline src/style.css]: `::after`

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+  box-sizing: border-box;
+}
+
+.nav-item {
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-of-type(1) {
+  margin-right: 0;
+}
```
Comment (danheim) [inline src/style.css]: Instead two selectors you can create only one:
```suggestion
.nav-item:not(:first-child) {
  // code
}
```

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,72 @@
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
```
Comment (danheim) [inline src/style.css]: Header has 100% width by default and body padding is 0.
It is not necessary here.

File: src/style.css (hunk 11 above)
Comment (PastushenkoVik) [reply src/style.css]: .nav-item:not(:first-child)  - select all without first. It is error

Comment (PastushenkoVik) [reply src/style.css]: Thanks. I did not immediately understand

File: src/index.html [hunk 13]
```diff
@@ -9,11 +9,59 @@
...
+  <title>Moyo</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+  <header>
+    <a href="#" class="logo-box">
+      <img src="./images/logo.png" alt="MOYO">
```
Comment (danheim) [inline src/index.html]: Screen reader reads this like `moyo`. What does it mean?
`Product photo` will be better.

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+}
+
+header {
```
Comment (danheim) [inline src/style.css]: Style classes, not tags

**Pull request closed without merging**